src/spdx2/ui/SpdxTwoCommaSeparatedValuesAgentPlugin.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ Copyright (C) 2021 Orange Author: Piotr Pszczola <piotr.pszczola@orange.com>
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+ SPDX-License-Identifier: GPL-2.0
+
+ */
+
+namespace Fossology\SpdxTwo;
+
+use Fossology\Lib\Plugin\AgentPlugin;
+
+/**
+ * @class SpdxTwoTagValueAgentPlugin
+ * @brief Add multiple uploads to SPDX2 report in Tag:Value format
```

Comment:
The comment looks wrong to me. Do you wanted to say "SPDX2 report in CSV format"?